docs/man/rpm-man-template.scd

```diff
@@ -124,10 +131,18 @@ On success, 0 is returned, a non-zero failure code otherwise.
 
 # EXAMPLES
 Add at least one example for each major functionality of the command.
-
-```
-rpmcmd --do-this package-1.0-1.noarch.rpm
-```
+Command first in bold, then description, similar to option descriptions.
+If command output is included in the description, use indented literal block.
```

Comment:
Just noting here that we may need to reconsider the indented literal block, due to the html converters that don't seem to grok that very well (in my testing), but that can be fixed later.

Reply:
Oh, right, those. Well, lets worry about it when we cross that bridge, it's a minor detail in the end.